app/views/help/cookies.html.haml

```diff
@@ -4,21 +4,112 @@
+        = t('cookies.section_functional.table.heading_two')
+      %th.govuk-table__header{scope: "col"}
+        = t('cookies.section_functional.table.heading_three')
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_functional.table.row_one_a')
+      %td.govuk-table__cell
+        = t('cookies.section_functional.table.row_one_b')
+      %td.govuk-table__cell
+        = t('cookies.section_functional.table.row_one_c')
+
+%h2.govuk-heading-l= t('cookies.section_analytics.section_heading')
+
+%p.govuk-body= t('cookies.section_analytics.paragraph_one')
+
+%p.govuk-body= t('cookies.section_analytics.paragraph_two')
 
 %ul.govuk-list.govuk-list--bullet
-  %li= t('cookies.section_two.list_item_one')
-  %li= t('cookies.section_two.list_item_two')
-  %li= t('cookies.section_two.list_item_three')
+  %li= t('cookies.section_analytics.list_item_one')
+  %li= t('cookies.section_analytics.list_item_two')
+  %li= t('cookies.section_analytics.list_item_three')
 
-%h2.govuk-heading-l= t('cookies.section_three.section_heading')
+%p.govuk-body= t('cookies.section_analytics.paragraph_three')
 
-%p.govuk-body= t('cookies.section_three.paragraph_one')
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('cookies.section_analytics.table.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{scope: "col"}
+        = t('cookies.section_analytics.table.heading_one')
+      %th.govuk-table__header{scope: "col"}
+        = t('cookies.section_analytics.table.heading_two')
+      %th.govuk-table__header{scope: "col"}
+        = t('cookies.section_analytics.table.heading_three')
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_one_a')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_one_b')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_one_c')
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_two_a')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_two_b')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_two_c')
+    %tr.govuk-table__row
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_three_a')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_three_b')
+      %td.govuk-table__cell
+        = t('cookies.section_analytics.table.row_three_c')
 
-%h2.govuk-heading-l= t('cookies.section_four.section_heading')
+%h2.govuk-heading-l= t('cookies.section_adjust.section_heading')
 
-%p.govuk-body= t('cookies.section_four.paragraph_one')
+%p.govuk-body
+  = t('cookies.section_adjust.pre_link_text')
+  %span
+  = link_to t('cookies.section_adjust.link_text'), cookie_settings_path, class: 'govuk-link'
+  %span>.
```

Comment:
I would recommend improving this by storing all this content as a single `key: value` pair in the locales file and passing the href value as an argument, for example:

Locales = `link_text: '<p class="govuk-body">You can <a class="govuk-link" href="%{href}">adjust your cookie settings</a>.</p>'`

cookies.html.haml = `= t('cookies.section_adjust.link_text', href: cookie_settings_path)`

Reply:
Agreed, I've changed it thus:

locales = 
```link_text_html: 'You can <a class="govuk-link" href="%{href}">adjust your cookie settings</a>.'```
haml = 
```
%p.govuk-body
  = t('cookies.section_adjust.link_text_html', href: cookie_settings_path)
```